src/app/app.component.html

```diff
@@ -0,0 +1,14 @@
+<main class="main">
+    <div class="stopwatch">
+      <span class="hours">{{ showHours }}</span>
+      <p class="colon">:</p>
+      <span class="minutes">{{ showMinutes }}</span>
+      <p class="colon">:</p>
+      <span class="seconds">{{ showSeconds }}</span>
```

Comment:
we can use single value with exact pipe to transform data instead of doing it manually